totes

```diff
@@ -119,6 +119,17 @@ handle_user_input() {
     n | N)
       totes "$@" # propagate arguments
       ;;
+    e | E)
+      if [[ -z "$EDITOR" ]]; then
+        printf "%s\n" "No value for \$EDITOR is set, so unable to edit the requested file."
```

Comment:
nit: since there's not actually a variable here you could do it all in a single string

Reply:
yes, will do